addons/dragging/CfgEventHandlers.hpp

```diff
@@ -32,6 +32,16 @@ class Extended_Init_EventHandlers {
             init = QUOTE(_this call DFUNC(initObject));
         };
     };
+    class RoadCone_F {
+        class ADDON {
+            init = QUOTE(_this call DFUNC(initObject));
+        };
+    };
+    class RoadCone_L_F {
```

Comment:
This class is insie `ReammoBox_F` class which already has XEH in this file, no need to have it twice.